src/lib.rs

```diff
@@ -2146,6 +2146,31 @@ fn ensure_libclang_is_loaded() {
 #[cfg(not(feature = "runtime"))]
 fn ensure_libclang_is_loaded() {}
 
+/// Error type for rust-bindgen.
+#[derive(Debug)]
```

Comment:
Nit: Derive more relevant traits (`Clone`, `PartialEq`, etc.)

Reply:
I have done in 2188742. Thanks.